### PR TITLE
Added 6.0 feature upgrade matrix

### PR DESCRIPTION
**Pull request by eric-schneider**

### CODE REVIEW

File: modules/install/pages/upgrade-strategy-for-features.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+| None
+
+| Scorch indexing system (Full Text Search)
+| Cluster
+| Offline/Online
+| Direct
+| Yes*
+| *FTS nodes running 6.0 use the Scorch indexing system by default for new indexes.
+You can continue to use previous indexes in 6.0, but they will not use Scorch.
+| To upgrade previous indexes to use Scorch, simply recreate them using the new index type option.
```
Comment (amarantha-k) [inline modules/install/pages/upgrade-strategy-for-features.adoc]: Add a link to the FTS create index page?

File: modules/install/pages/upgrade-strategy-for-features.adoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+|===
+
+[#feature-mixed-mode]
+† _Mixed Mode in this context refers to only the nodes that are running the indicated service. So if 'No' is marked in the Mixed Mode column of a given feature, then all nodes that are running that service must be upgraded to the same version in order to enable that feature._
+
+[#upgrade-ipv6]
+== Migrating Data from an IPv4 Cluster to an IPv6 Cluster
```
Comment (amarantha-k) [inline modules/install/pages/upgrade-strategy-for-features.adoc:43]: Version number update from 5.5 to 6.0?

**Review by amarantha-k — CHANGES_REQUESTED** [review]
Added inline comments

File: modules/install/pages/upgrade-strategy-for-features.adoc (hunk 1 above)
Comment (eric-schneider) [reply modules/install/pages/upgrade-strategy-for-features.adoc]: Done

File: modules/install/pages/upgrade-strategy-for-features.adoc (hunk 2 above)
Comment (eric-schneider) [reply modules/install/pages/upgrade-strategy-for-features.adoc:43]: I assume this is regarding the 5.5 references for IPv6. It's not specific to 6.0, but right now there isn't a good home for this information, so I left it in the same place it was for 5.5. Also there is a good chance these procedures will change soon.

**Review by amarantha-k — APPROVED** [review]

**Pull request merged**